source/channels.yaml

```diff
@@ -343,6 +343,39 @@
         '500':
           description: Could not retrieve the channel member, the user is not a member of the channel or the channel does not belong to the team
 
+  '/teams/{team_id}/channels/{channel_id}/members/ids':
+      post:
+        tags:
+          - channels
+        summary: Get channel members by IDs
+        description: Get a map of channel member objects for the specified channel and user IDs.
+        parameters:
+          - name: team_id
+            in: path
+            description: Team ID of the member to return
+            required: true
+            type: string
+          - name: channel_id
+            in: path
+            description: The channel ID
+            required: true
+            type: string
+          - in: body
+            name: body
+            description: A list of user IDs
+            required: true
+            schema:
+              type: array
+              items:
+                type: string
+        responses:
+          '200':
+            description: Members retreived successfully
+            schema:
+              type: object
```

Comment:
Good question! It looks like the equivalent method for TeamMembers returns an Array in the code, but claims to return a Map in the API reference.

I'll change this one to an array too, unless you think it makes more sense to change things to a map here?